source/Reference.js

```diff
@@ -2,4 +2,14 @@ module.exports.GAME_WIDTH = 230
 module.exports.GAME_HEIGHT = 130
 module.exports.JOIN_GAME_TEXT = "PRESS TO JOIN"
 module.exports.JUNK_FREQUENCY_RANGE = { lower: 5, upper: 10}
-module.exports.TIME_TO_DESPAWN = 5 * 1000
\ No newline at end of file
+module.exports.TIME_TO_DESPAWN = 5 * 1000
+
+module.exports.ControlScheme = {
+    WASD: 0,
+    NUM_PAD: 1,
+    keys: {
+        0: {up: "W", down: "S", left: "A", right: "D", fire: "<space>", inUse: false},
+        1: {up: "<num-8>", down: "<num-2>", left: "<num-4>", right: "<num-6>", fire: "<num-0>", inUse: false}
+    }
+}
+module.exports.controlTypeCount = 2
```

Comment:
Hmmmm; I like that we're indexing into `ControlScheme` using the player number, like 0 or 1. But why do we allow the option to index by strings like WASD or NUM_PAD? Let's make that a child of the `ControlScheme` JSON.